src/main/java/dk/kb/ds/cumulus/export/Configuration.java

```diff
@@ -115,11 +122,15 @@ public static String getCollection(){
     public static String getOutputFile(){
         return instance().outputFile;
     }
-
     public static String getType(){
         return instance().type;
     }
-
+    public static String getLimited(){
+        return instance().limited;
+    }
+    public static String getCounter(){
```

Comment:
When we know the type of a specific property (in this case that `counter`is an integer) have a first-claass support for that property (the method `getCounter`), the method should take care of parsing and return the value as the type that is expected: `public static int getCounter()`.

Reply:
Corrected in new commit

Reply:
Great! Thanks.